docs/examples/guide/workers/weather05.py

```diff
@@ -21,7 +22,7 @@ def compose(self) -> ComposeResult:
 
     async def on_input_changed(self, message: Input.Changed) -> None:
         """Called when the input changes"""
-        self.update_weather(message.value)
+        self.update_weather(quote(message.value))
```

Comment:
I think a better place for the `quote` would be the `update_weather` method itself.

Reply:
Even better spotted 🚀

Reply:
@danschwarz I went ahead and moved the `quote` for you so we can deploy the docs fix ASAP!
I hope you don't mind 😄